src/main/scala/ai/eto/rikai/sql/spark/RikaiSparkSessionExtensions.scala

```diff
@@ -73,21 +74,13 @@ private class MlPredictRule(val session: SparkSession)
     }
   }
 
-  @enableIf(c =>
-    c.classPath.exists(
-      _.getPath.matches(".*spark-catalyst_2\\.\\d+-3\\.2\\..*")
-    )
-  )
+  @enableIf(classpathMatches(".*spark-catalyst_2\\.\\d+-3\\.2\\..*".r))
```

Comment:
Can it check against "not spark 3.1"?

This enableIf does not work with 3.3+ right?  3.3 is being voted at the moment.

Reply:
the latest regex check against "not spark 3.0 or 3.1", please review again

Reply:
The latest regex check against "not spark 3.1 or 3.0", please review again